src/common/colorspaces_inline_conversions.h

```diff
@@ -193,9 +193,11 @@ static inline float cbrta_halleyf(const float a, const float R)
   return b;
 }
 
+/* this produces a crash in Windows with gcc -Ofast #8025
```

Comment:
Just a question as I have not followed all the discussion. Why do we disable this on all platforms if it break Windows ? Or is the bug on all platforms but only visible/reproducible on Windows ?

Reply:
This  is a good question, I have the issue only with Windows / gcc, but it seems that performance penalization is small because the code vectorization was poor anyhow. I leave the last word to you   more expert devs. Eventually we can restrict this change to Windows only

Reply:
Coverted to draft because I noticed that lab_f() is called in many modules that do create any problem, so disabling simd there can't be right, even if restricted only to Windows.
I have to investigate a bit more

Reply:
Maybe the -Ofast option is the culprit although i don't what detail triggers the bug.

Reply:
most likely.
In many modules, `lab_f()` is called from `dt_XYZ_to_Lab()` so we have even deeper nesting and no problems.
Instead `#pragma GCC optimize ("-Ofast")` is only used in `rcd_demosaic.c` and `dual_demosaic.c`

Reply:
And this is not the first time we had issues with `-Ofast`. I would propose to remove this from this file. Maybe that's because such optimization is better to be used globally ? As we have a set of inline and not inline who knows what an optim in one file can do... So if ok please remove `-Ofast` and we will revisit this whole issue in another PR.

Reply:
Especially when one read:

```
-Ofast
    Disregard strict standards compliance.  ...
```

Let's keep away of this.

Reply:
Please see last commit.

- reverted change in lab_f()
- disabled gcc -Ofast only for the problematic function blend_images() in dual_demosaic.c and only for Windows.

This seems to me the fix with the smallest impact.
If other issues will come out with -Ofast in Windows, we will have to disable it completely